ed448/src/lib.rs.orig

```diff
@@ -0,0 +1,201 @@
+#![no_std]
```

Comment:
Think you forgot to remove this

Reply:
sorry, bitten by jujutsu's autosnapshot feature.